tests/cards/action.test.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { CardActions, Button } from "libs/";
+
+describe("components/CardActions", () => {
```

Comment:
it should be `"cards/CardActions"` (that will be easier to locate the source file if the test fails eventually)